### PR TITLE
WIP - Pages to delete and redirect

### PR DESCRIPTION
**Pull request by rachelwhitton**
## Summary
Deletes pages, updates impacted links, some consolidations 

## Post-Merge Redirects Required 
* [x] [Pantheon Overview](https://docs.pantheon.io/overview) => [Pantheon Docs Home](https://docs.pantheon.io/)
* [x] [Account Management at Pantheon](https://docs.pantheon.io/manage) => [Account Management at Pantheon](https://docs.pantheon.io/guides/account-mgmt)
* [x] [Custom Upstreams](https://docs.pantheon.io/custom-upstreams) => [Custom Upstreams on Pantheon: Introduction](https://docs.pantheon.io/guides/custom-upstream)
* [x] [Autopilot Product Overview](https://docs.pantheon.io/autopilot) => [Pantheon Autopilot: Introduction](https://docs.pantheon.io/guides/autopilot)
* [x] [Terminus Overview](https://docs.pantheon.io/terminus-overview) => [Terminus Guide: Introduction](https://docs.pantheon.io/termius) 
* [x] [New Relic Product Overview](https://docs.pantheon.io/newrelic) => [New Relic Performance Monitoring on Pantheon: Introduction](https://docs.pantheon.io/guides/new-relic)
* [x] [Sign Up for an Account](https://docs.pantheon.io/sign-up) => [Get Started with Pantheon: Sign Up](https://docs.pantheon.io/guides/getstarted/signup)
* [x] [WordPress on Pantheon: Pre-GA](https://docs.pantheon.io/guides/wordpress-composer/pre-ga) => [Integrated Composer: Introduction](https://docs.pantheon.io/guides/integrated-composer)
* [x] [The Pantheon.yml Configuration File](https://docs.pantheon.io/pantheon-yml-overview) => [Pantheon YAML Configuration Files](https://docs.pantheon.io/pantheon-yml)
* [x] [Pantheon Drupal (Composer Managed) Upstream](https://docs.pantheon.io/drupal-composer-managed) => [Pantheon Release Notes](https://docs.pantheon.io/release-notes/1/)
* [x] [Pantheon WordPress Upstream](https://docs.pantheon.io/wordpress) => [Pantheon Release Notes](https://docs.pantheon.io/release-notes/1/)
* [x] [Configuring Visual Studio Code for Pantheon](https://docs.pantheon.io/visual-studio-code) => [Local Development on Pantheon: Configuring Visual Studio Code for Pantheon](https://docs.pantheon.io/guides/local-development/visual-studio-code)
* [x] [Configuring JetBrains PhpStorm IDE with Drupal on Pantheon](https://docs.pantheon.io/drupal-phpstorm) => [Local Development on Pantheon: Configure PhpStorm for Drupal](https://docs.pantheon.io/guides/local-development/drupal-phpstorm)
* [x] [Introduction to Terminus](https://docs.pantheon.io/terminus-demo) => [Terminus Guide: Introduction](https://docs.pantheon.io/termius)
* [x] [Enterprise Billing Center Product Overview](https://docs.pantheon.io/enterprise-billing-center) => [Enterprise Billing Center: Introduction](https://docs.pantheon.io/guides/enterprise-billing-center)
* [x] [Billing Docs](https://docs.pantheon.io/billing) => [Account Management at Pantheon](https://docs.pantheon.io/guides/account-mgmt)
* [x] [Adding a Site to Pantheon](https://docs.pantheon.io/add-site) => [Get Started with Pantheon: Add a Site](https://docs.pantheon.io/guides/getstarted/addsite)
* [x] [Create a New CMS Site](https://docs.pantheon.io/add-site-dashboard) => [Get Started with Pantheon: Add a Site](https://docs.pantheon.io/guides/getstarted/addsite)
* [x] [Pantheon Workshops](https://docs.pantheon.io/workshops) => [Pantheon WebOps Certification](https://docs.pantheon.io/certification)
* [x] [Google Domain Configuration](https://docs.pantheon.io/google) => [DNS Host-specific DNS Instructions](https://docs.pantheon.io/dns-providers)
* [x] [Deployment Infrastructure Upgrade](https://docs.pantheon.io/platform-upgrade) => [Monthly Platform Maintenance](https://docs.pantheon.io/monthly-maintenance)
* [x] [Using Terminus to Create and Update Drupal Sites on Pantheon](https://docs.pantheon.io/terminus-drupal-site-management) => [Create a Drupal Site From the Command Line Using Terminus and Drush](https://docs.pantheon.io/guides/drush/drupal-commandline)
* [x] [Guided WordPress Migrations](https://docs.pantheon.io/migrate-wordpress) => [Migrate Sites to Pantheon Using Guided Migration: Migrate](https://docs.pantheon.io/guides/guided/migrate)
* [x] [Drupal 10 on Pantheon](https://docs.pantheon.io/drupal-10) => [Supported Drupal Versions](https://docs.pantheon.io/supported-drupal)
* [x] [Running WordPress and Drupal as a Backend API](https://docs.pantheon.io/headless) => [Front-End Sites on Pantheon](https://docs.pantheon.io/guides/decoupled)
* [x] [Install Cygwin on Windows](https://docs.pantheon.io/cygwin-windows) => [Local Development on Pantheon: Install Cygwin on Windows](https://docs.pantheon.io/guides/local-development/cygwin-windows)
* [x] [Migrate a Composer-Managed Drupal Site from Another Platform: Prepare](https://docs.pantheon.io/guides/manual-d8-composer-to-d8/prepare) => [Migrate a Composer-Managed Drupal Site from Another Platform: Prepare](https://docs.pantheon.io/guides/drupal-unhosted-composer/prepare)
* [x] [Migrate a Composer-Managed Drupal Site from Another Platform: Add Contrib and Custom Code](https://docs.pantheon.io/guides/manual-d8-composer-to-d8/contrib-custom) => [Migrate a Composer-Managed Drupal Site from Another Platform: Add Contrib and Custom Code](https://docs.pantheon.io/guides/drupal-unhosted-composer/contrib-custom)
* [x] [Migrate a Composer-Managed Drupal Site from Another Platform: Deploy to Dev](https://docs.pantheon.io/guides/manual-d8-composer-to-d8/deploy-dev) => [Migrate a Composer-Managed Drupal Site from Another Platform: Deploy to Dev](https://docs.pantheon.io/guides/drupal-unhosted-composer/deploy-dev)
* [x] [Migrate a Composer-Managed Drupal Site from Another Platform: Add Your Database](https://docs.pantheon.io/guides/manual-d8-composer-to-d8/database) => [Migrate a Composer-Managed Drupal Site from Another Platform: Add Your Database](https://docs.pantheon.io/guides/drupal-unhosted-composer/database)
* [x] [Migrate a Composer-Managed Drupal Site from Another Platform: Upload Your Files](https://docs.pantheon.io/guides/manual-d8-composer-to-d8/files) => [Migrate a Composer-Managed Drupal Site from Another Platform: Upload Your Files](https://docs.pantheon.io/guides/drupal-unhosted-composer/files)
* [x] [Migrate a Composer-Managed Drupal Site from Another Platform: Deploy](https://docs.pantheon.io/guides/manual-d8-composer-to-d8/deploy-live) => [Migrate a Composer-Managed Drupal Site from Another Platform: Deploy](https://docs.pantheon.io/guides/drupal-unhosted-composer/deploy-live)
* [x] [Migrate a Composer-Managed Drupal Site from Another Platform: Troubleshooting](https://docs.pantheon.io/guides/manual-d8-composer-to-d8/troubleshooting) => [Migrate a Composer-Managed Drupal Site from Another Platform: Troubleshooting](https://docs.pantheon.io/guides/drupal-unhosted-composer/troubleshooting)
* [x] [Migrate a Composer-Managed Drupal Site from Another Platform: Introduction](https://docs.pantheon.io/guides/manual-d8-composer-to-d8) => [Migrate a Composer-Managed Drupal Site from Another Platform: Introduction](https://docs.pantheon.io/guides/drupal-unhosted-composer/)

### CODE REVIEW

File: source/content/crisis-response-upstream.md [hunk 1]
```diff
@@ -74,7 +74,7 @@ These plugins were selected to improve the content creation and moderation proce
 
   If not, use your email address for the username and click on **Forgot Password**. Be sure to set a strong password.
 
-1. After the site is configured correctly in the Dev environment, you can [create the Test and Live environments](/add-site-dashboard#create-the-test-environment), copying your code and the initial database to each.
+1. After the site is configured correctly in the Dev environment, you can [create the Test and Live environments](/guides/getstarted/addsite/#create-the-test-environment), copying your code and the initial database to each.
```
Comment (stevector) [inline source/content/crisis-response-upstream.md:77]: This heading does not exist in the new page

File: source/content/guides/account-mgmt/traffic/01-introduction.md [hunk 2]
```diff
@@ -16,7 +16,7 @@ audience: [sysadmin]
 product: [--]
 integration: [--]
 ---
-Access Metrics through the Live tab of the Site Dashboard once a [Live environment has been initialized](/add-site-dashboard#create-the-test-environment). The number of unique visits displayed in Pantheon’s Site Dashboard determines the traffic Pantheon will apply for evaluating use on your site under your pricing plan. The Site Dashboard also includes other information you may use to project future traffic, including number of pages served.
+Access Metrics through the Live tab of the Site Dashboard once a [Live environment has been initialized](/guides/getstarted/addsite/#create-the-live-environment). The number of unique visits displayed in Pantheon’s Site Dashboard determines the traffic Pantheon will apply for evaluating use on your site under your pricing plan. The Site Dashboard also includes other information you may use to project future traffic, including number of pages served.
```
Comment (stevector) [inline source/content/guides/account-mgmt/traffic/01-introduction.md:19]: This heading does not exist

File: source/content/crisis-response-upstream.md (hunk 1 above)
Comment (stevector) [reply source/content/crisis-response-upstream.md:77]: NM, I was looking at docs.pantheon.io, not the PR preview.

File: source/content/guides/account-mgmt/traffic/01-introduction.md (hunk 2 above)
Comment (stevector) [reply source/content/guides/account-mgmt/traffic/01-introduction.md:19]: NM, I was looking at docs.pantheon.io, not the PR preview.

**Comment by rachelwhitton** [comment]
@stevector is it a problem that a deleted guide is still present here in this file? https://github.com/pantheon-systems/documentation/blob/redirects-new-nav/src/components/omniSidebarNav/testfixtures/allGuides.testfixture.js#L4053-L4148 

I tried to remove these lines locally but had issues with pushing the change because the git hook errors: 
Screenshot 2024-12-04 at 12 01 42 PM

So I just left them in the file as they are, hoping that's not an issue?

**Comment by stevector** [comment]
> hoping that's not an issue?

I don't think it's an issue because this is a test fixture (sample data) that is used to confirm the behavior/logic of the helper functions that reshape the data from what comes out of graphQL to what goes into the menu components.

**Review by stevector — APPROVED** [review]

**Pull request merged**